README.md

```
@@ -105,68 +117,190 @@ Meet our wonderful team comprised of Product Managers, Designers, Tech Leads, En
+```bash
+git clone https://github.com/Hack4Impact-UMD/journey-home.git
+```
+Then use the `cd` command to enter the project directory and then install the project dependencies.
+```bash
+cd journey-home
+npm install
+```
+> ❗❗❗ **IMPORTANT**: There are some environment secrets that the project needs to run that are stored in a `.env` file. If you've been provided a `.env` file to use, just put this directly in your project directory. If not, copy the `env.example` file to `.env` and then configure the secrets in there.
+
+For development, we will be using the Firebase emulator and some other Firebase CLI tools that are from the `firebase-tools` package. This package will be installed globally on your system so you can run the commands from anywhere.
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)